lib/oga/xml/lexer.rb

```diff
@@ -161,7 +147,7 @@ def lex
           tokens << [type, value, line]
         end
 
-        reset
+        reset_native
```

Comment:
This can be removed as well, `reset_native` should only be called in `initialize`.